x-pack/platform/plugins/shared/fleet/server/services/epm/packages/get.ts

```diff
@@ -800,3 +811,56 @@ export async function getPackageAssetsMap({
     throw error;
   }
 }
+
+/**
+ * Return assets agent template assets map for package policies operation
+ */
+export async function getAgentTemplateAssetsMap({
```

Comment:
should this be covered with unit tests?

Reply:
Yes it seems a good candidate, I will add one